src/canary-sign-in-with-ipv.js

```diff
@@ -49,115 +50,39 @@ const basicCustomEntryPoint = async () => {
   log.info(`Empty OTP code bucket (${bucketName})`);
   await emptyOtpBucket(bucketName, phoneNumber);
 
-  let page = await synthetics.getPage();
-  const navigationPromise = page.waitForNavigation({
-    waitUntil: "networkidle0",
-  });
+  const page = await synthetics.getPage();
 
-  // TODO: remove ref to sandpit - this is only temp so we can point at integration for now
-  if (CANARY_NAME.includes("integration") || isSandpitJourney()) {
+  // Currently, sandpit canaries need to be run against the integration backend
+  if (CANARY_NAME.includes("integration") || CANARY_NAME.includes("sandpit")) {
```

Comment:
Ah thinking about this it's a bit more complex since the name of the canary is like "integration-smoke-in" rather than just integration, so the succint version you mention wouldn't work. We could definitely do something like this by trimming the string etc or doing something else, but I think then it becomes more complex than currently so planning on leaving this.

Reply:
Hadn't clocked this - in that case yeah ignore me; agreed it's not worth the extra faff :)